javascript/javascript1/week3/note_taking.js

```diff
@@ -0,0 +1,76 @@
+// NOnoN0nOYes (Note taking app)
+const notes = [];
+
+// Save a note
+function saveNote(content, id) {
+    let noteObject = {};
+    noteObject.content = content;
+    noteObject.id = id;
+    notes.push(noteObject);
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+console.log(notes);
+
+// Get a note
+function getNote(id) {
+    let noteIndex;
+    let noteContent = "";
+    if((id === undefined) || (typeof(id) !== 'number')) {
+        noteContent = "Please enter a valid id first!!!";
+    } else {
+        for(let i = 0; i <= notes.length; i++) {
+            if(noteIndex !== undefined) {
```

Comment:
why do you need this condition here?

Reply:
I don't remember now :) I also don't think, I need it here. Thanks.